### PR TITLE
Dev 0 login page

### PR DESCRIPTION
**Pull request by marissadarnell**

### CODE REVIEW

**Comment by marissadarnell** [comment]
image

image

File: src/components/authentication/login.jsx [hunk 1]
```diff
@@ -1,22 +1,130 @@
...
+/* changes width of input fields */
+const Credentials = styled(Form)`
+  width: 100%;
+`;
+
+const SignIn = styled.button`
+  border-radius: 5px;
+  padding: 6px 0px; 
+  width: 48%;
+  font-family: Roboto;
```
Comment (ctperry0301) [inline src/components/authentication/login.jsx:55]: change font-family to fontFamily to get the style to apply.

File: src/components/authentication/login.jsx [hunk 2]
```diff
@@ -1,22 +1,130 @@
...
+  display: flex;
+  font-family: Roboto;
+  margin-left: auto; 
+  text-decoration: none;
+  color: #6C6B6B;
+  &:hover{
+    text-decoration: none;
+    color: black;
+  }
 `;
```
Comment (ctperry0301) [inline src/components/authentication/login.jsx:100]: Can you add please add these Styles to the Image. (The image would stretch when i ran localhost on my phone)

const StyledImage = styled(Image)`
  width: 100%;
  height: 100%;
`;

**Review by ctperry0301 — CHANGES_REQUESTED** [review]
Looks really good so far, there are a few small changes but it looks and works well. 

Also, I dont think you use the image in the public folder, so you should be good to remove it.

File: src/components/authentication/login.jsx (hunk 1 above)
Comment (marissadarnell) [reply src/components/authentication/login.jsx:55]: For me, fontFamily doesn't change the font. Do you still want me to change it?

Comment (ctperry0301) [reply src/components/authentication/login.jsx:60]: Thats really weird, because the opposite thing happens for me. Maybe you could add them both? I

**Review by ctperry0301 — APPROVED** [review]
Looks good! Great work Marissa. (Don't worry about the font thing for now, I can look into it next week)

**Pull request merged**